test/DynamoCoreWpfTests/CodeBlockNodeTests.cs

```diff
@@ -182,5 +182,15 @@ public void TestSyntaxHighlightRuleForDigits()
             string[] expected = new string[] { "-2468.2342E+04", "34534.345345", "23423", "-98.7", "0", "10", "2", "-555" };
             Assert.IsTrue(expected.SequenceEqual(actual));
         }
+
+        [Test, RequiresSTA]
+        public void TestFunctionMultipleBlocksDefaultParametersDeleteFirst()
+        {
+            RunCommandsFromFile("DeleteCrashCommands.xml", (commandTag) =>
+            {
+                // NOTE: Nothing needs to be tested here other than that this test does not crash
```

Comment:
does this need to be wrapped in an assert does not throw?

Reply:
This is probably true, I will add this